src/accounts/accountModel.java

```diff
@@ -0,0 +1,121 @@
+package accounts;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author aswecker
+ */
+public class accountModel implements Initializable {
+
+    private static final int ID = 1;
+    private static final int First_Name = 1;
+    private static final int Last_Name = 1;
+    private static final int Username = 1;
+    private static final int Password = 1;
+    
+    private static final String COMMA_DELIMITER = ",";
+    private static final String NEW_LINE = "\n";
+    private static final String FILE_HEADER = "Id,First_Name,Last_Name,Username,Password";
+    
+  
+
+   
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+        addUser();
+        readUser();
+    }
+
+    // Add a user
+    public void addUser() {
+        User u = new User(1, "Test", "User", "TestUser", "password");
+        User u1 = new User(2, "John", "Doe", "JohnDoe", "password");
+        List<User> Users = new ArrayList<User>();
+
+        Users.add(u);
+        Users.add(u1);
+
+        File file = new File("data/Accounts.csv");
+        FileWriter writer = null;
+
+        try {
+            writer = new FileWriter(file);
+            writer.append(FILE_HEADER);
+            writer.append(COMMA_DELIMITER);
+            writer.append(NEW_LINE);
+
+            for (User usr : Users) {
+                writer.append(String.valueOf(usr.Getuserid()));
+                writer.append(COMMA_DELIMITER);
+                writer.append(usr.Getfname());
+                writer.append(COMMA_DELIMITER);
+                writer.append(usr.Getlname());
+                writer.append(COMMA_DELIMITER);
+                writer.append(usr.Getusername());
+                writer.append(COMMA_DELIMITER);
+                writer.append(usr.Getpassword());
+                writer.append(NEW_LINE);
+            }
+        } catch (Exception e) {
+            System.out.println("Error writing CSV file");
+            e.printStackTrace();
+        } finally {
+            try {
+                writer.flush();
+                writer.close();
+            } catch (IOException e) {
+                System.out.println("Error closing the fileWriter");
+                e.printStackTrace();
+            }
+        }
+    }
+
+    /**
+     * Read the file.
+     */
+    public void readUser() {
```

Comment:
This is useful as a private method, but for the purpose of authentication we will need to be able to give this method a userID, then have a User object or null (invalid user) returned. There also needs to be a way to authenticate the user's password. This could be a method in the User class or maybe this class could handle it. Either way is fine, but we need one of them in order for the user to be able to log in.